Analyze foreign giving to UVA. Compare UVA to appropriate peer institutions and the market overall. Propose a strategy for finding new sources of foreign money and provide a quantitative analysis.

In [ ]:
import numpy as np
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt

df = pd.read_csv('ForeignGifts_edu.csv',low_memory=False)
df.head()

In [22]:
u_totals =df.groupby('Institution Name')['Foreign Gift Amount'].sum()
median = u_totals.median()
print(median)


6486791.5


In [27]:
competitors = [
    'University of Virginia',
    'University of North Carolina - Chapel Hill',
    'Duke University',
    'University of Notre Dame',
    'Virginia Polytechnic Institute & State University',
]
gift_totals = (
    df[df['Institution Name'].isin(competitors)]
    .groupby('Institution Name')
    ['Foreign Gift Amount'].sum()
    .sort_values(ascending=False)
)
print(gift_totals)

Institution Name
Duke University                                      343699498
University of Notre Dame                              46652439
University of North Carolina - Chapel Hill            41292544
University of Virginia                                22189238
Virginia Polytechnic Institute & State University     11792062
Name: Foreign Gift Amount, dtype: int64


In [26]:
gift_counts = (
    df[df['Institution Name'].isin(competitors)]
    .groupby('Institution Name')
    .size()
    .sort_values(ascending=False)
)

print(gift_counts)

Institution Name
University of North Carolina - Chapel Hill           286
Duke University                                      145
University of Notre Dame                             122
University of Virginia                                98
Virginia Polytechnic Institute & State University     24
dtype: int64


While UVA ($22,189,238) is well ahead of the median college in terms of foreign gifts received by school ($6,486,791.5), when compared to its peer institutions in the top of the ACC, they are not doing well, receiving not only less gift dollars, but less total gifts as well.

In [28]:
average_gift = (
    df[df['Institution Name'].isin(competitors)]
    .groupby('Institution Name')['Foreign Gift Amount']
    .mean()
    .sort_values(ascending=False)
)

print(average_gift)

Institution Name
Duke University                                      2.370341e+06
Virginia Polytechnic Institute & State University    4.913359e+05
University of Notre Dame                             3.823970e+05
University of Virginia                               2.264208e+05
University of North Carolina - Chapel Hill           1.443795e+05
Name: Foreign Gift Amount, dtype: float64


This tells us that UVA has an average higher than UNC, which means they get more money than UVA because they get significantly more gifts. UVA should focus their efforts on acquiring more gifts.

In [30]:
uva = df[df['Institution Name'] == 'University of Virginia']

country_counts = uva['Country of Giftor'].value_counts()

print(country_counts)

Country of Giftor
ENGLAND            31
ISRAEL             12
BANGLADESH         10
SWEDEN             10
FINLAND             7
KOREA               6
FRANCE              5
THE NETHERLANDS     4
TANZANIA            3
GUERNSEY            3
SINGAPORE           2
SWITZERLAND         2
JAPAN               1
BERMUDA             1
RWANDA              1
Name: count, dtype: int64


In [31]:
country_totals = uva.groupby('Country of Giftor')['Foreign Gift Amount'].sum().sort_values(ascending=False)

print(country_totals)

Country of Giftor
BANGLADESH         4228589
SWEDEN             4007552
TANZANIA           2776447
SWITZERLAND        2000000
ENGLAND            1950666
GUERNSEY           1500000
FINLAND            1008384
BERMUDA             900000
KOREA               862787
SINGAPORE           805000
THE NETHERLANDS     725670
ISRAEL              623655
RWANDA              284135
JAPAN               265970
FRANCE              250383
Name: Foreign Gift Amount, dtype: int64


England has given UVA the most gifts. Despite this, Bangladesh has given UVA the most money

In [ ]:

uva_countries = set(df[df['Institution Name'] == 'University of Virginia']['Country of Giftor'])

competitor_country_sets = [
    set(df[df['Institution Name'] == school]['Country of Giftor'])
    for school in competitors
]

countries_in_all_competitors = set.intersection(*competitor_country_sets)

missing_from_uva = countries_in_all_competitors - uva_countries

print(missing_from_uva)

set()
